c2corg_ui/static/js/documentediting.js

```diff
@@ -374,6 +394,23 @@ app.DocumentEditingController.prototype.handleMapFeatureChange_ = function(
 
 
 /**
+ * It doesn't associate, it just pushes it into associations array.
+ * @private
+ */
+app.DocumentEditingController.prototype.pushDocToAssociations_ = function() {
+  var doctype = this.urlParams_['doctype'];
+  var docid = this.urlParams_['docToAssociateId'];
+  $.get(this.apiUrl_ + '/' + doctype + '/' + docid, function(doc) {
```

Comment:
right. Where can i get the full list of possible arguments for filtering? 
(and btw can you tell me where is the full doc with steps for indexing elastic search? it is not https://github.com/c2corg/v6_api, maybe somewhere deeper in the directory but i don't remember)